src/client/cockpit-client

```diff
@@ -55,7 +55,7 @@ def prctl(*args):
         raise Exception('prctl() failed')
 
 
-prctl.SET_PDEATHSIG = 1
+prctl.SET_PDEATHSIG = 1  # type: ignore[attr-defined]
```

Comment:
This really belongs in `systemd_ctypes`...

Reply:
Hmm, we could at least stop littering `prctl` with it and define it as a normal global, like we do in src/cockpit/transports.py

Reply:
Yes.  That would also be an improvement.  This is copypasta and I think I fixed the other copy of it because mypy was complaining...

Reply:
Done.